flict/flictlib/project/reader.py

```diff
@@ -17,7 +19,11 @@ def __init__(self, project_file, project_dirs):
 
     @staticmethod
     def dependencies_license(package):
-        licenses = [Project.package_license(dep) for dep in package.get('dependencies', [])]
+        licenses = []
+        for dep in package.get('dependencies', []):
+            if 'license' not in dep:
+                raise FlictError(ReturnCodes.RET_INVALID_PROJECT, f'Dependency does not contain "license": {dep}')
+            licenses.append(f" ( {dep['license']}) ")
```

Comment:
Why is this reverted back to its old state?

Reply:
Just to elaborate: The former code does exactly the same as the new one with the only difference being the error message starting with `Dependency` instead of package.

Reply:
new state back with the coming push